source/WindowScroller/WindowScroller.css

```diff
@@ -0,0 +1,3 @@
+.disabled > * {
+  pointer-events: none !important;
```

Comment:
> The first disables pointer events only for immediate children. The latter disables for all descendants.

Well, yeah, but [`pointer-events` is inherited](http://codepen.io/anon/pen/pNWejN), which I guess is what I was counting on there—analogous to WindowScroller in master counting on elements to inherit it from `document.body`.

>I wonder if it would be more appropriate to approach this by passing a prop (eg disablePointerEvents) to Grid telling it to disable pointer events.

That works, but it _is_ different than the existing effect. WindowScroller in current `master` disables pointer-events on `document.body`, e.g. all the stuff inside the scroll container. My CSS (while obviously not being feasible) accomplishes exactly that for arbitrary scroll containers. What you’re suggesting would narrow the scope of what gets `pointer-events: none` down to just the grid/list, not the whole scroll container. That seems like a reasonable trade-off to me, but I don’t have the same prior knowledge as you about how `pointer-events` effects performance of this whole system. If you’re ok with that trade-off, I can move forward with implementing it.